gtsfm/frontend/correspondence_generator/keypoint_aggregator/keypoint_aggregator_dedup.py

```diff
@@ -15,11 +15,17 @@
 
 
 class KeypointAggregatorDedup(KeypointAggregatorBase):
-    """Keypoint aggregator with de-duplication."""
+    """Keypoint aggregator with de-duplication of keypoints within each image."""
 
-    def __init__(self) -> None:
-        """Initialize global variables"""
+    def __init__(self, nms_merge_radius: float = 3) -> None:
+        """Initialize global variables.
+
+        Args:
+            nms_merge_radius: Radius (in pixels) to use when merging detections within the same view (image).
+                Note that tracks are merged, not suppressed.
+        """
```

Comment:
I really think taking NMS out of the name would reduce any  confusion about supression. I think just `merge_radius` is fine, and then you wouldn't have to put the note.

Reply:
Also, have you done any testing with respect to the merge radius? Is 3 pixels the best value you saw?